blog/models.py

```diff
@@ -48,9 +51,9 @@ def get_featured_image_path(self, filename):
     author = models.ForeignKey(User, on_delete=models.CASCADE, related_name='Post', verbose_name='Author')
     content = RichTextField()
     date = models.DateField(default=date.today)
-    featured_image = models.ImageField(default='', verbose_name='Featured Image', upload_to=get_featured_image_path)
+    featured_image = ProcessedImageField(default='', verbose_name='Featured Image', upload_to=get_featured_image_path, validators=[validate_file_size], **processed_image_field_specs)
     category = models.ForeignKey(Category, on_delete=models.SET_NULL, verbose_name='Category', null=True, blank=True)
-    tags = models.ManyToManyField(Tag, verbose_name='Tag', blank=True)
+    tags = models.ManyToManyField(Tag,verbose_name='Tag', blank=True)
```

Comment:
its always adviced to reformat your code (all the files) before committing, this ensures proper spacing and indentation are in place automatically.